apps/backend/src/users/users.controller.spec.ts

```diff
@@ -113,7 +113,8 @@ describe('UsersController Unit Tests', () => {
       expect.assertions(1);
 
       const createdUser = await usersController.create(
-        CREATE_USER_DTO_TEST_OBJ_2
+        CREATE_USER_DTO_TEST_OBJ_2,
+        {user: adminUser}
```

Comment:
Does this mean that the test suite always runs with registration disabled now?

Reply:
Registration is not disabled for the test suite, the problem here is that the only way I seem to be able to get the user's information from the request (to check if they are an admin) is to use `JwtAuthGuard`, however, using that means requests have to be authenticated. So I added `POST /authn/signup` which is the non-authenticated version of `POST /users`, which will be disabled when configured in .env.